src/@types/event.ts

```diff
@@ -53,6 +53,7 @@ export enum EventType {
     CallReject = "m.call.reject",
     CallSelectAnswer = "m.call.select_answer",
     CallNegotiate = "m.call.negotiate",
+    CallSDPStreamMetadataChanged = "org.matrix.call.sdp_stream_metadata_changed",
```

Comment:
Best practice is to *send* only the prefixed version but support *receiving* both, that way once it's unprefixed, old clients will work seamlessly with new ones.